{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2024-38875",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "An issue was discovered in Django 4.2 before 4.2.14 and 5.0 before 5.0.7. urlize and urlizetrunc were subject to a potential denial of service attack via certain inputs with a very large number of brackets.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2024-38875",
				"url": "https://www.suse.com/security/cve/CVE-2024-38875"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1227590 for CVE-2024-38875",
				"url": "https://bugzilla.suse.com/1227590"
			},
			{
				"category": "external",
				"summary": "Advisory link for SUSE-SU-2024:2545-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2024-July/018995.html"
			},
			{
				"category": "external",
				"summary": "Advisory link for SUSE-SU-2024:2577-1",
				"url": "https://lists.suse.com/pipermail/sle-updates/2024-July/036097.html"
			},
			{
				"category": "external",
				"summary": "Advisory link for openSUSE-SU-2024:0251-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/OU4KXNSFOQVRSGL2OQCMRA3EFMPZEGEU/"
			}
		],
		"title": "SUSE CVE CVE-2024-38875",
		"tracking": {
			"current_release_date": "2025-01-04T01:01:09Z",
			"generator": {
				"date": "2024-07-10T03:25:03Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2024-38875",
			"initial_release_date": "2024-07-10T03:25:03Z",
			"revision_history": [
				{
					"date": "2024-07-10T03:25:03Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2024-07-11T03:14:10Z",
					"number": "3",
					"summary": "Current version"
				},
				{
					"date": "2024-07-18T02:38:36Z",
					"number": "4",
					"summary": "Current version"
				},
				{
					"date": "2024-07-19T02:39:47Z",
					"number": "5",
					"summary": "Current version"
				},
				{
					"date": "2024-07-20T02:52:59Z",
					"number": "6",
					"summary": "Current version"
				},
				{
					"date": "2024-07-23T02:17:02Z",
					"number": "7",
					"summary": "Current version"
				},
				{
					"date": "2024-07-26T03:12:44Z",
					"number": "8",
					"summary": "Current version"
				},
				{
					"date": "2024-08-19T01:33:43Z",
					"number": "9",
					"summary": "Current version"
				},
				{
					"date": "2025-01-01T00:58:58Z",
					"number": "10",
					"summary": "Current version"
				},
				{
					"date": "2025-01-04T01:01:09Z",
					"number": "11",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "11"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Linux Enterprise Module for Package Hub 15 SP6",
								"product": {
									"name": "SUSE Linux Enterprise Module for Package Hub 15 SP6",
									"product_id": "SUSE Linux Enterprise Module for Package Hub 15 SP6",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:packagehub:15:sp6"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Leap 15.5",
								"product": {
									"name": "openSUSE Leap 15.5",
									"product_id": "openSUSE Leap 15.5",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:leap:15.5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Leap 15.6",
								"product": {
									"name": "openSUSE Leap 15.6",
									"product_id": "openSUSE Leap 15.6",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:leap:15.6"
									}
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django",
								"product": {
									"name": "python-Django",
									"product_id": "python-Django",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python3-Django",
								"product": {
									"name": "python3-Django",
									"product_id": "python3-Django"
								}
							},
							{
								"category": "product_version",
								"name": "python3-Django-2.0.7-150000.1.20.1",
								"product": {
									"name": "python3-Django-2.0.7-150000.1.20.1",
									"product_id": "python3-Django-2.0.7-150000.1.20.1"
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django-5.0.7-2.1",
								"product": {
									"name": "python310-Django-5.0.7-2.1",
									"product_id": "python310-Django-5.0.7-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:5.0.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.14-1.1",
								"product": {
									"name": "python310-Django4-4.2.14-1.1",
									"product_id": "python310-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django",
								"product": {
									"name": "python311-Django",
									"product_id": "python311-Django",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django-4.2.11-150600.3.3.1",
								"product": {
									"name": "python311-Django-4.2.11-150600.3.3.1",
									"product_id": "python311-Django-4.2.11-150600.3.3.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:4.2.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django-5.0.7-2.1",
								"product": {
									"name": "python311-Django-5.0.7-2.1",
									"product_id": "python311-Django-5.0.7-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:5.0.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.14-1.1",
								"product": {
									"name": "python311-Django4-4.2.14-1.1",
									"product_id": "python311-Django4-4.2.14-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django-5.0.7-2.1",
								"product": {
									"name": "python312-Django-5.0.7-2.1",
									"product_id": "python312-Django-5.0.7-2.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:5.0.7:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.14-1.1",
								"product": {
									"name": "python312-Django4-4.2.14-1.1",
									"product_id": "python312-Django4-4.2.14-1.1"
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-4.2.11-150600.3.3.1 as component of SUSE Linux Enterprise Module for Package Hub 15 SP6",
					"product_id": "SUSE Linux Enterprise Module for Package Hub 15 SP6:python311-Django-4.2.11-150600.3.3.1"
				},
				"product_reference": "python311-Django-4.2.11-150600.3.3.1",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Package Hub 15 SP6"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-Django-2.0.7-150000.1.20.1 as component of openSUSE Leap 15.5",
					"product_id": "openSUSE Leap 15.5:python3-Django-2.0.7-150000.1.20.1"
				},
				"product_reference": "python3-Django-2.0.7-150000.1.20.1",
				"relates_to_product_reference": "openSUSE Leap 15.5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-4.2.11-150600.3.3.1 as component of openSUSE Leap 15.6",
					"product_id": "openSUSE Leap 15.6:python311-Django-4.2.11-150600.3.3.1"
				},
				"product_reference": "python311-Django-4.2.11-150600.3.3.1",
				"relates_to_product_reference": "openSUSE Leap 15.6"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django-5.0.7-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django-5.0.7-2.1"
				},
				"product_reference": "python310-Django-5.0.7-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.14-1.1"
				},
				"product_reference": "python310-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django-5.0.7-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django-5.0.7-2.1"
				},
				"product_reference": "python311-Django-5.0.7-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.14-1.1"
				},
				"product_reference": "python311-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django-5.0.7-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django-5.0.7-2.1"
				},
				"product_reference": "python312-Django-5.0.7-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.14-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
				},
				"product_reference": "python312-Django4-4.2.14-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django as component of SUSE Linux Enterprise Module for Package Hub 15 SP6",
					"product_id": "SUSE Linux Enterprise Module for Package Hub 15 SP6:python311-Django"
				},
				"product_reference": "python311-Django",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Package Hub 15 SP6"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of SUSE Linux Enterprise Module for Package Hub 15 SP6",
					"product_id": "SUSE Linux Enterprise Module for Package Hub 15 SP6:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Package Hub 15 SP6"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-Django as component of openSUSE Leap 15.5",
					"product_id": "openSUSE Leap 15.5:python3-Django"
				},
				"product_reference": "python3-Django",
				"relates_to_product_reference": "openSUSE Leap 15.5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django as component of openSUSE Leap 15.5",
					"product_id": "openSUSE Leap 15.5:python-Django"
				},
				"product_reference": "python-Django",
				"relates_to_product_reference": "openSUSE Leap 15.5"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2024-38875",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-38875"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 4.2 before 4.2.14 and 5.0 before 5.0.7. urlize and urlizetrunc were subject to a potential denial of service attack via certain inputs with a very large number of brackets.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"known_affected": [
					"SUSE Linux Enterprise Module for Package Hub 15 SP6:python-Django",
					"openSUSE Leap 15.5:python-Django"
				],
				"recommended": [
					"SUSE Linux Enterprise Module for Package Hub 15 SP6:python311-Django-4.2.11-150600.3.3.1",
					"openSUSE Leap 15.5:python3-Django-2.0.7-150000.1.20.1",
					"openSUSE Leap 15.6:python311-Django-4.2.11-150600.3.3.1",
					"openSUSE Tumbleweed:python310-Django-5.0.7-2.1",
					"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python311-Django-5.0.7-2.1",
					"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
					"openSUSE Tumbleweed:python312-Django-5.0.7-2.1",
					"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-38875",
					"url": "https://www.suse.com/security/cve/CVE-2024-38875"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1227590 for CVE-2024-38875",
					"url": "https://bugzilla.suse.com/1227590"
				},
				{
					"category": "external",
					"summary": "Advisory link for SUSE-SU-2024:2545-1",
					"url": "https://lists.suse.com/pipermail/sle-security-updates/2024-July/018995.html"
				},
				{
					"category": "external",
					"summary": "Advisory link for SUSE-SU-2024:2577-1",
					"url": "https://lists.suse.com/pipermail/sle-updates/2024-July/036097.html"
				},
				{
					"category": "external",
					"summary": "Advisory link for openSUSE-SU-2024:0251-1",
					"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/OU4KXNSFOQVRSGL2OQCMRA3EFMPZEGEU/"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Linux Enterprise Module for Package Hub 15 SP6:python311-Django-4.2.11-150600.3.3.1",
						"openSUSE Leap 15.5:python3-Django-2.0.7-150000.1.20.1",
						"openSUSE Leap 15.6:python311-Django-4.2.11-150600.3.3.1",
						"openSUSE Tumbleweed:python310-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"SUSE Linux Enterprise Module for Package Hub 15 SP6:python311-Django-4.2.11-150600.3.3.1",
						"openSUSE Leap 15.5:python3-Django-2.0.7-150000.1.20.1",
						"openSUSE Leap 15.6:python311-Django-4.2.11-150600.3.3.1",
						"openSUSE Tumbleweed:python310-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python310-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python311-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python311-Django4-4.2.14-1.1",
						"openSUSE Tumbleweed:python312-Django-5.0.7-2.1",
						"openSUSE Tumbleweed:python312-Django4-4.2.14-1.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-07-09T16:00:10Z",
					"details": "important"
				}
			],
			"title": "CVE-2024-38875"
		}
	]
}
